### PR TITLE
Django example

### PR DESCRIPTION
**Pull request by lfvilella**
Adding simple [Django](https://www.djangoproject.com/) example.

### CODE REVIEW

File: django/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: '3'
```
Comment (glours) [inline django/docker-compose.yml:1]: Can you be more specific on the version and use `3.7`

**Review by glours — CHANGES_REQUESTED** [review]
Thanks @lfvilella for your contribution.
Can you also update the main [README.md](https://github.com/docker/awesome-compose/blob/master/README.md) to add your sample in the Single service samples section please?

**Review by glours — APPROVED** [review]
LGTM

**Comment by glours** [comment]
@lfvilella Thank you very much for the pull request 🙏

**Review by mandervrik — COMMENTED** [review]
Sorry for the late

**Pull request merged**